### ISSUE TITLE
package argument of adapter.dispatch is deprecated

### ISSUE THREAD

**Issue by emmyoop** [dbt-labs/dbt-labs-experimental-features#15]
`adapter.dispatch` is deprecating the `packages` argument.  In v1.0.0 the use of `packages` will raise an exception.  Issue for this change: [#3895](https://github.com/dbt-labs/dbt/issues/3895)

https://github.com/dbt-labs/dbt-labs-experimental-features/blob/84e676bb0721d32e206e9f7e31def6ce4321513e/materialized-views/macros/default/adapters.sql#L1-L15

Fixing it should just consist of replacing both `adapter-dispatch` lines with:
```
{{ return(adapter.dispatch('create_materialized_view_as', 'dbt_labs_materialized_views')(relation, sql, config)) }}
```

**Comment by jtcohen6** [comment]
Thanks @emmyoop! It's been a while since I've updated the code here, but I was thinking of taking a look soon anyway...

**Comment by jtcohen6** [comment]
Resolved by #16
